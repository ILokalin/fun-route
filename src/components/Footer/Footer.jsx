import React from "react"


export default function () {
  
  return  (
    <div className="footer">
      <a className="footer__link" href="https://github.com/ILokalin/fun-rout" rel="noopener noreferrer" target="_blank">Ilya Lokalin. Test of React.JS 2019
      </a>
    </div>
  )
}