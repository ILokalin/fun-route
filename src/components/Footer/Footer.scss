.footer {
  text-align: center;


  .footer__link {
    color: $WhiteElement-color;
    text-decoration: none;
    transition: all 0.3s linear;

    &:hover {
      color: $ActiveElement-color;
    }
  }
}