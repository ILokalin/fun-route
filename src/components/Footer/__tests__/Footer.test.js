import React from 'react';
import ReactTestUtils from 'react-dom/test-utils';
import rendrer from 'react-test-renderer';

import Footer from 'components/Footer';


describe('unit', () => {
  
  it('render snap', () => {
    const component = rendrer.create(<Footer />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    })
})
