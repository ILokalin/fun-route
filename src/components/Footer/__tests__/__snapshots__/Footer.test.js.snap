// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit render snap 1`] = `
<div
  className="footer"
>
  <a
    className="footer__link"
    href="https://github.com/ILokalin/fun-route"
    rel="noopener noreferrer"
    target="_blank"
  >
    Ilya Lokalin. Test of React.JS 2019
  </a>
</div>
`;
